app/controllers/CyaCreateProfileController.scala

```diff
@@ -41,15 +36,16 @@ class CyaCreateProfileController @Inject() (
   getData: DataRetrievalAction,
   requireData: DataRequiredAction,
   checkProfile: ProfileCheckAction,
-  dataCleansingService: DataCleansingService,
   val controllerComponents: MessagesControllerComponents,
   view: CyaCreateProfileView,
   traderProfileConnector: TraderProfileConnector,
+  implicit val dataCleansingService: DataCleansingService,
```

Comment:
I don't see the advantage of making this implicit/I think we would benefit from doing the cleansing in the controller to make the functionality less hidden

Reply:
See above, pop me a message if you like, I'm concerned that by doing it in the controllers then we're just repeating behaviour